scripts/run-ui.sh

```diff
@@ -6,11 +6,18 @@ rm -r $PWD/fed-ui
 mkdir $PWD/fed-ui
 
 # start bitcoind on regtest in the background
+export FM_BITCOIND_RPC="http://bitcoin:bitcoin@127.0.0.1:18443"
 bitcoind -regtest -daemon -fallbackfee=0.0004 -txindex -server -rpcuser=bitcoin -rpcpassword=bitcoin
 
 # start guardians
 for ((ID = 0; ID < $FM_FED_SIZE; ID++)); do
   mkdir $PWD/fed-ui/mock-$ID
+  p2p_port=$((10000 + 10000 * $ID))
+  api_port=$((10001 + 10000 * $ID))
+  export FM_BIND_P2P=127.0.0.1:$p2p_port
+  export FM_P2P_URL=ws://127.0.0.1:$p2p_port
```

Comment:
Can you change this to `export FM_P2P_URL=fedimint://127.0.0.1:$p2p_port`

We use a custom protocol so `fedimint://` is a better prefix than `ws://`.

Reply:
I've changed that now. Does this also apply to `FM_API_URL`?

Reply:
Ah OK, thanks for clarifying that.

Sure, I'll create a separate PR for the placeholder and link it to the issue you've just created (#1744).